runtime/Cargo.toml

```diff
@@ -21,6 +21,8 @@ deno_fetch = "0.129.0"
 deno_url = "0.105.0"
 deno_web = "0.136.0"
 deno_webidl = "0.105.0"
+lassie = "0.3.0"
```

Comment:
Neat. I didn't know there is a Rust client!

Reply:
It's a thin Rust wrapper embedding the original Go Lassie, I started the project three weeks ago :)

https://github.com/filecoin-station/rusty-lassie